app/about.md

```diff
@@ -6,13 +6,13 @@ Welcome to the dashboard for the University of Arizona [Campus Living Lab](http:
 
 This innovative campus initiative was designed to transform our green stormwater infrastructure (GSI) features located across campus into an interactive learning environment for students, researchers, and partners from various disciplines.
 
-![Photo Credit: Ryan Hunt, BIO5 Institute](CampusLivingLab(12-01-23)-0185.jpg){width="640"}
+![Photo Credit: Ryan Hunt, BIO5 Institute](CampusLivingLab(12-01-23)-0185.jpg){style="text-align: center;" width="662"}
```

Comment:
text-align only works for text.  There are some other ideas to try here: https://dev.to/bdavidxyz/markdown-center-image-39j1